classes/class-bsfrt-readtime.php

```diff
@@ -212,7 +225,7 @@ public function bsf_rt_init_frontend() {
 
 		// Show Reading time Conditions.
 		if ( $this->bsf_rt_get_option( 'bsf_rt_show_read_time' ) && 'none' !== $this->bsf_rt_options['bsf_rt_position_of_read_time'] ) {
-			if ( in_array( 'bsf_rt_single_page', $this->bsf_rt_options['bsf_rt_show_read_time'] ) && is_singular() ) {//PHPCS:ignore:WordPress.PHP.StrictInArray.MissingTrueStrict
+			if ( in_array( 'bsf_rt_single_page', $this->bsf_rt_options['bsf_rt_show_read_time'] ) && is_singular() && ! $this->bsf_rt_styles_loaded_flag ) {//PHPCS:ignore:WordPress.PHP.StrictInArray.MissingTrueStrict
```

Comment:
I think for now don't handle duplicate reading time rendering, As this condition has above & below title rendering too, there might be some user's who might me using it.